usr.sbin/adduser/adduser.sh

```diff
@@ -479,6 +506,20 @@ get_homeperm() {
 	fi
 }
 
+# get_zfs_home
+#	Determine if homeprefix is located on a ZFS filesystem and if
+#	so, enable ZFS home dataset creation.
+#
+get_zfs_home() {
```

Comment:
This seems like it would override a configured `Zflag=no`, but I'm also not sure I understand the logic; `zfs list` would exit 0 if it found a prefix, and `-z "${zfs_homeprefix}` would flip Zflag on if we didn't detect a prefix.

There's an additional caveat that using `zfs` will trigger a load of zfs.ko on purely ufs systems; I'd recommend bailing out before calling it if `kldstat -q -m zfs` indicates that zfs isn't there

Reply:
Oh, right, Zflag means "no zfs dataset"... the name is definitely getting me here.